### ISSUE TITLE
[TC]: Check keyboard navigation

### ISSUE THREAD

**Issue by Florina87** [TCs-for-Association-projects/Peviitor_TC#94]
### Epic

Epic5: Footer Navigation & Structure  (#16)

### User Story

E5 · US3: US3 : As a user with accessibility needs, I want to navigate the footer using assistive technologies so that I can access all links without barriers (#19)

### Summary

TC-The website can be navigated only by keyboard

### Description

The website can be used without a mouse, using only the keyboard. 

### Preconditions

- Website is accessible and loaded successfully
- Keyboard is connected and functioning properly.

### Test Data

Keyboard keys :
- Tab
- Shift + Tab
- Enter
- Space
- Arrow keys
- Esc

### Testing Type

Accessibility testing

### Website Section

Footer

### Test Environment

Production

### Environment Details

OS: Windows 10 Pro
Browser: Chrome 147.0
Resolution: 1920x1080
Device: Laptop Dell Inspiron 15 5000 series

### Cross-platform Coverage

- [ ] Should be run across multiple operating systems
- [ ] Should be run across multiple browsers

### Test Steps

**Step 1.** [Open the website peviitor.ro in browser]
- [ ] Expected: [The page is open]

**Step 2.** [Press Tab to navigate through interactive elements on the page]
- [ ] Expected: [Each interactive element on the page is in the correct order, and the focused element is clearly visible]

**Step 3.** [Verify the focus moves sequentially and visibly to each interactive element]
- [ ] Expected: [Keyboard focus moves sequentially and visibly to each interactive element on the page when navigating using the Tab key]

### Confirm before submitting

- [x] Steps are clear and repeatable.
- [x] Epic and User Story match.
